### ISSUE TITLE
wechaty-puppet-padpro 群里面拿不到非好友的微信号？

### ISSUE THREAD

**Issue by chelsh** [Chatie/wechaty#1711]
wechaty-puppet-padpro 群里面拿不到非好友的微信号？是否可以支持下啊，我们现在很需要啊，因为用户不喜欢填手机号码，我们想在每条信息上自动附上群用户的微信号的，现在非好友拿不到微信号，很痛苦啊。

**Comment by lijiarui** [comment]
Sorry, we won't support this feature.

**Comment by BoltDoggy** [comment]
群里也拿不到非好友的性别 / 省份 / 城市

**Comment by lijiarui** [comment]
@BoltDoggy  Yes, you are right, and we won't support this at the near future.
